src/fdb5/toc/TocHandler.cc

```diff
@@ -1340,7 +1340,7 @@ std::vector<Index> TocHandler::loadIndexes(const Catalogue& catalogue, bool sort
                     s >> offset;
                     s >> type;
                     LOG_DEBUG(debug, LibFdb5) << "TocRecord TOC_INDEX " << path << " - " << offset << std::endl;
-                    tocindexes[entry.seqNo] = new TocIndex(s, catalogue, entry.datap->header_.serialisationVersion_,
+                    tocindexes[entry.seqNo] = new TocIndex(s, entry.datap->header_.serialisationVersion_,
```

Comment:
could you remove `catalogue` from 1- lambda, and 2- `::loadIndexes()` .. also from `enumerateMasked` ?

Reply:
not this PR but can add `threads.reserve(nthreads);` above ?